include/SimulationConfigurationNetworker.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include <proto/SimulationConfiguration.pb.h>
+#include <utils/Publisher.hpp>
+#include <utils/Subscriber.hpp>
+
+#include <functional>
```

Comment:
Doesn't seem to be used

Reply:
It is used in line 19 because of the `std::function`